### PR TITLE
examples: msr-events intercepts 6 MSRs by default

### PR DESCRIPTION
**Pull request by Wenzel**
rework msr-events example

* intercepts 6 MSRs by default
* can specify custom set of MSRs

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ libc = { version = "0.2.58", optional = true }
 xenctrl = { git = "https://github.com/Wenzel/xenctrl", optional = true }
 xenstore = { git = "https://github.com/Wenzel/xenstore", optional = true }
 xenforeignmemory = { git = "https://github.com/Wenzel/xenforeignmemory", optional = true }
-kvmi = { git = "https://github.com/Wenzel/kvmi", optional = true }
+kvmi = { git = "https://github.com/Wenzel/kvmi", rev = "dd10135a27bb3658d399dfb5477299ca0f4baeac", optional = true }
```
Comment (rageagainsthepc) [inline Cargo.toml:27]: Why do you pin this commit?

**Comment by rageagainsthepc** [comment]
So you want to pass raw integers instead of register names? Why? Imho thats not handy at all, especially not for an example.

**Comment by Wenzel** [comment]
Because it doesn't limit this example to the 6 hardcoded registers we have defined, which are now intercepted by default.

The end goal is to use the x86 crate with their msr definitions, so you can pass (any) string again.

I need to open an issue to ask them to split it into a separate crate, bevause it requires nightly rust now.

**Comment by rageagainsthepc** [comment]
But those are all the registers we are able to intercept right now. If we want to intercept more, we would need to change the code of kvmi as well.

**Comment by rageagainsthepc** [comment]
My bad, I think I mixed something up. It's perfectly possible to intercept arbitrary MSRs. Sorry about the confusion.

File: Cargo.toml (hunk 1 above)
Comment (Wenzel) [reply Cargo.toml:27]: because using `master` broke the CI

**Pull request merged**